main.tf

```diff
@@ -22,6 +22,14 @@ resource "azurerm_api_management_api" "apim_api" {
   import {
     content_format = var.api_content_format
     content_value  = var.api_content_value
+
+    dynamic "wsdl_selector" {
+      for_each = (var.api_content_format == "wsdl") ? [1] : []
```

Comment:
The [documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/api_management_api#wsdl_selector) indicates that wsdl_selector block can only be specified when the content_format is "wsdl" or "wsdl-link".  Can you add "wsdl-link" to the check as well?

Reply:
done